dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/identifiable/IdentifierTypeServiceImpl.java

```diff
@@ -105,6 +110,11 @@ public Map<String, String> updateIdentifierTypeCache() throws ServiceException {
                   Collectors.toConcurrentMap(
                       IdentifierType::getNamespace, IdentifierType::getPattern));
     } catch (RepositoryException e) {
+      if (e.getMessage().contains("relation \"identifiertypes\" does not exist")) {
+        LOGGER.warn(
+            "The identifier type cache could not be initialised, because the corresponding relation in the DB does not yet exist - please restart the application after the migration have run.");
```

Comment:
```suggestion
            "The identifier type cache could not be initialised, because the corresponding relation in the DB does not yet exist - please restart the application after the migrations have run.");
```